src/plopp/functions/inspector.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from .common import require_interactive_backend, preprocess
+from .figure import figure
+from ..core import input_node, widget_node, Node
+from ..core.utils import coord_as_bin_edges
+from ..widgets import PointsTool
+
+from scipp import Variable, scalar
+from scipp.typing import VariableLike
```

Comment:
I importing without namespace good style? As this is not part of scipp anymore we should maybe just `import scipp as sc`?

Reply:
I don't know, i'm also importing things from `ipywidgets` like that. e.g. `from ipywidgets import VBox`.
I thought it is cheaper than importing the whole of `ipywidgets`, but maybe it makes no difference in practise because it still has to run through the whole `__init__.py`?

Reply:
I guess it is a question of style, and we have never made a clear decision on that.